src/components/JoinUs/Work.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { CompensatedCol, CompensatedRow } from '../grid'
+import { H3, Paragraph } from '../Typography'
+import colorLuminance from 'color-luminance'
+
+import { Section, Separator, TitleAndBody, TitleAndList } from './elements'
+
+const MAX_CASE_STUDIES = 3
+
+/* Utility function to convert an HEX color into the RGB format */
+const hexToRgb = hex => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16)
+      }
+    : null
+}
+
+/* Returns the luminance (in a 0-255 range) of a color in the HEX format */
+const getColorLuminance = hexColor => {
+  const rgbValue = hexToRgb(`#${hexColor}`)
+
+  return colorLuminance(rgbValue.r, rgbValue.g, rgbValue.b)
+}
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+
+const CaseStudy = ({ title, lightText, ...props }) => (
```

Comment:
Asked Tom & we've agreed that this component needs its own Zeplin documentation. Especially because Beatriz is looking to use this on another page. So it might be useful to move this out into the Common folder now